extensions/wikia/AdEngine/js/video/playwire.js

```diff
@@ -12,11 +12,17 @@ define('ext.wikia.adEngine.video.playwire', [
 		return '//config.playwire.com/' + publisherId + '/videos/v2/' + videoId + '/zeus.json';
 	}
 
-	function inject(configUrl, parent, vastUrl) {
-		var script = doc.createElement('script');
+	function inject(params) {
+		var configUrl = params.configUrl,
+			container = params.container,
+			vastUrl = params.vastUrl,
+			slotName = params.slotName,
+			width = params.width,
```

Comment:
What if it's not passed? It will break all creatives that are not using creative template?

+ alphabetical order